pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dev_dependencies:
   dart_dev: "^1.0.0"
   dart_style: "^0.2.1"
   dartdoc: "^0.8.0"
+  logging: "^0.11.0"
```

Comment:
Oh wow, there were no real dependencies previously, I actually feel slightly bad for adding the first one!